tests/tests_suite_test.go

```diff
@@ -25,6 +25,11 @@ type Cmd struct {
 	CommandLineString string
 }
 
+type App struct {
```

Comment:
This is the right solution--we need to keep URL separate from name. (We may also want to store whether it's a buildpack or Dockerfile or `deis pull` app in the `App` struct at some point, so we can vary the `deployApp` strategy and know whether to use `cmd` or `web`--see code `TODO:`s in #53).